tests/infra/network.py

```diff
@@ -36,7 +36,7 @@
 JOIN_TIMEOUT = 40
 
 # If it takes a node n seconds to call an election, how long should we wait for an election to succeed?
-DEFAULT_TIMEOUT_MULTIPLIER = 3
+DEFAULT_TIMEOUT_MULTIPLIER = 15
```

Comment:
Aligning everything on a single timeout, because:

1. It's easier to reason about
2. The existing diversity of values seem to be an accident of history
3. We don't pay for this most of the time, unless we're negatively testing the lack of convergence (there is one such use, and we lower this to 3), or a test fails.